### PR TITLE
Update loadjs and load vendor and app bundles in parallel.

### PR DESCRIPTION
**Pull request by bearMountain**
- fetch in parallel
- load in series

### CODE REVIEW

File: packages/gluestick/src/renderer/helpers/getAssetsLoader.js [hunk 1]
```diff
@@ -16,24 +16,16 @@ module.exports = function getAssetsLoader(
...
 
     var loadVendorThenEntry = function() {
-      loadjs('${vendorBundle}', {
-        success: loadEntryPoint,
-        error: function() { throw new Error('Failed to load ${vendorBundle}'); },
+      loadjs(['${vendorBundle}', '${entryPointBundle}'], {
+        error: function() { throw new Error('Failed to load ${vendorBundle} or ${entryPointBundle}'); },
+        async: false,
```
Comment (sethbattin) [inline packages/gluestick/src/renderer/helpers/getAssetsLoader.js]: 👍

File: packages/gluestick/src/renderer/helpers/getAssetsLoader.js [hunk 2]
```diff
@@ -16,24 +16,16 @@ module.exports = function getAssetsLoader(
...
 
     var loadVendorThenEntry = function() {
-      loadjs('${vendorBundle}', {
-        success: loadEntryPoint,
-        error: function() { throw new Error('Failed to load ${vendorBundle}'); },
+      loadjs(['${vendorBundle}', '${entryPointBundle}'], {
+        error: function() { throw new Error('Failed to load ${vendorBundle} or ${entryPointBundle}'); },
```
Comment (zamotany) [inline packages/gluestick/src/renderer/helpers/getAssetsLoader.js]: I believe you can get the bundle name from first argument of that function.

File: packages/gluestick/src/renderer/helpers/getAssetsLoader.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ module.exports = function getAssetsLoader(
 
     var loadVendorThenEntry = function() {
       loadjs(['${vendorBundle}', '${entryPointBundle}'], {
-        error: function(pathsNotFound) { throw new Error('Failed to load ${vendorBundle} or ${entryPointBundle}. Missing paths count: ${pathsNotFound.length}. First missing path: ${pathsNotFound[0]}'); },
+        error: function(pathsNotFound) { throw new Error('Failed to load ${vendorBundle} or ${entryPointBundle}. Missing paths count: ' + pathsNotFound.length + '. First missing path: ' + pathsNotFound[0]); },
```
Comment (toddw) [inline packages/gluestick/src/renderer/helpers/getAssetsLoader.js]: What is this Missing paths count stuff for?

Comment (toddw) [reply packages/gluestick/src/renderer/helpers/getAssetsLoader.js]: What about `throw new Error('Failed to load ' + pathsNotFound.join(', '))` ?

**Comment by bearMountain** [comment]
- `pathsNotFound` param added
- error message string simplified
- vendor order loading tested on chrome, firefox, and IE 11

**Review by sethbattin — APPROVED** [review]
👍 still looks good

**Pull request merged**